scripts/github_release.rb

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env ruby
+
+require 'fileutils'
+require 'pathname'
+
+require 'octokit'
+
+BUILD_SH = Pathname(__FILE__).+('../../build.sh').expand_path
+VERSION = `sh '#{BUILD_SH}' get-version`.strip
+RELEASE = "v#{VERSION}"
+
+BUILD = ARGV[0] ? Pathname(ARGV[0]).expand_path : BUILD_SH.parent + 'build'
+OBJC_ZIP = BUILD + "realm-objc-#{VERSION}.zip"
+SWIFT_ZIP = BUILD + "realm-swift-#{VERSION}.zip"
+CARTHAGE_ZIP = BUILD + 'Carthage.framework.zip'
+
+FileUtils.cp SWIFT_ZIP, CARTHAGE_ZIP
```

Comment:
The Carthage zip isn't just a copy of the Swift zip. Here's how I make the Carthage zip with bash:

```
mkdir -p Carthage/Build/Mac Carthage/Build/iOS
mv realm-swift-0.93.1/ios/Realm.framework realm-swift-0.93.1/ios/RealmSwift.framework Carthage/Build/iOS
mv realm-swift-0.93.1/osx/Realm.framework realm-swift-0.93.1/osx/RealmSwift.framework Carthage/Build/Mac
zip --symlinks -r Carthage.framework.zip Carthage
```